.github/workflows/password-rules.yml

```diff
@@ -1,37 +1,40 @@
 name: Password Rules
 
 on:
-  pull_request:
-    types: [opened]
+  workflow_dispatch:
+  schedule:
+    - cron: '0 3 * * *' # every morning at 3am UTC
 
 jobs:
-  test:
-    runs-on: ubuntu-20.04
+  update:
+    name: Get latest password rules
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
           node-version-file: '.nvmrc'
-      - uses: actions/cache@v2
+      - name: Get latest rules
+        run: node ./packages/password/scripts/rules.js --write-rules-json
+      - name: Check for changes
+        id: changes
+        run: |
+          filesChanged=$(sh ./scripts/check-for-changes.sh)
+          if [ -z "$filesChanged" ]; then hasChanged="false"; else hasChanged="true"; fi
+          echo "hasChanged=$hasChanged" >> "$GITHUB_OUTPUT"
+        shell: bash {0}
+      - name: Get current date
+        id: date
+        run: |
+          echo "currentDate=$(date +'%Y-%m-%d')" >> "$GITHUB_OUTPUT"
+      - name: Create PR for updated rules
+        if: ${{ steps.changes.outputs.hasChanged == 'true' }}
+        uses: peter-evans/create-pull-request@88bf0de51c7487d91e1abbb4899332e602c58bbf
```

Comment:
I was going to question the use of this - but then I realised it's used in the autofill automation already :)

Reply:
@shakyShane Yeah, just copy-pasted the one we're using for the release flow 😁  -- I assume it's the version that's been checked and that we're good to use.